src/components/file/detailsFile/DrawerDetailsFile.tsx

```diff
@@ -10,13 +19,23 @@ import { IPCContact, IPCFile } from 'types/types';
 
 import colors from 'theme/foundations/colors';
 import getFileType from '../../../utils/getFileType';
+import formatDate from '../../../utils/formatDate';
+import formatFileSize from '../../../utils/formatFileSize';
```

Comment:
```suggestion
import getFileType from './utils/getFileType';
import formatDate from './utils/formatDate';
import formatFileSize from './utils/formatFileSize';
```
should work as we have set a `baseSrc` in `tsconfig.json`